### DISCUSSION TITLE
How to use extended scalars with dgs?

### DISCUSSION THREAD

**Discussion by jessepinkman9900** [Netflix/dgs-framework#247] [Q&A] [answered]
I would like to use https://github.com/graphql-java/graphql-java-extended-scalars with dgs. 
How do I register the scalar in dgs?

**Comment by srinivasankavitha** [comment, marked as answer]
You can add it a dependency and register the scalar manually as described here:
https://netflix.github.io/dgs/scalars/

In your schema file: 
```
scalar Long
```

For registering:
```
@DgsComponent
public class LongScalarRegistration {
    @DgsRuntimeWiring
    public RuntimeWiring.Builder addScalar(RuntimeWiring.Builder builder) {
        return builder.scalar(Scalars.GraphQLLong);
    }
}
```

That said, in the next release, you shouldn't need to manually register it. For now, you can use the above mechanism.

**Reply by jessepinkman9900** [reply]
when is the next release expected to come?

**Reply by timpamungkas** [reply]
seems like [my question](https://github.com/Netflix/dgs-framework/discussions/313)
